### PR TITLE
Update Landing Page copy

### PR DESCRIPTION
**Pull request by tomrf1**
Updates the copy in the landing page.
Also refactors the config in productCatalog.tsx to make it easier to reuse copy.

Screenshot 2024-10-31 at 15 28 09
Screenshot 2024-10-31 at 15 28 40

### CODE REVIEW

**Review by charleycampbell — APPROVED** [review]
⛴️

File: support-frontend/assets/pages/supporter-plus-landing/components/threeTierCard.tsx [hunk 1]
```diff
@@ -227,7 +227,7 @@ export function ThreeTierCard({
 			{isRecommended && !isUserSelected && (
 				<ThreeTierLozenge
 					subdue={isRecommendedSubdued}
-					title={promotion?.landingPage?.roundel ?? 'Recommended'}
+					title={promotion?.landingPage?.roundel ?? 'Highest impact'}
```
Comment (andrewHEguardian) [inline support-frontend/assets/pages/supporter-plus-landing/components/threeTierCard.tsx:230]: Do we want to test this change?

**Comment by charleycampbell** [comment]
Looks good to me! The only thing from the requirements _The rewards from All-access digital_ isn't in bold, should we check what they would like?

**Review by andrewHEguardian — APPROVED** [review]
Neat!

**Review by paul-daniel-dempsey — COMMENTED** [review]
neat refactor Sue & Tom

**Pull request merged**